tube-player/modules/10-Add-media-player/README.md

```diff
@@ -180,16 +180,17 @@ protected async override void OnNavigatingFrom(NavigatingCancelEventArgs e)
 }
 ```
 
-## Install additional NuGet packages for WASM and Skia.GTK
+## Configure Uno Platform Features
```

Comment:
We can also mention here that this step could be done previously in the Wizard by selecting "Media Player" in "Features" or by adding the parameter in the CLI (dotnet new ...). Need to check the parameter though.